yarn-project/sequencer-client/src/block_builder/solo_block_builder.test.ts

```diff
@@ -326,8 +323,12 @@ describe('sequencer/solo_block_builder', () => {
 
       processedTx.data.end.newL2ToL1Msgs = makeTuple(MAX_NEW_L2_TO_L1_MSGS_PER_TX, fr, seed + 0x300);
       processedTx.data.end.newContracts = [makeNewContractData(seed + 0x1000)];
-      processedTx.data.end.encryptedLogsHash = to2Fields(L2Block.computeKernelLogsHash(processedTx.encryptedLogs));
-      processedTx.data.end.unencryptedLogsHash = to2Fields(L2Block.computeKernelLogsHash(processedTx.unencryptedLogs));
+      processedTx.data.end.encryptedLogsHash = Fr.fromBufferReduce(
```

Comment:
`fromBufferReduce` also does the modulus remainder style reduction to the prime field